source/docs/contributing/translations.rst

```diff
@@ -0,0 +1,54 @@
+Translations
+============
+
+frc-docs supports translations using the web-based `Transifex <https://www.transifex.com>`__ utility. frc-docs supports Chinese (zh_CN), French Canada (fr_CA), Hebrew Israel (he_IL), Spanish Mexico (es_MX), and Turkish (tr_TR). Translators that are fluent in *both* English and one of the specified languages would be greatly appreciated to contribute to the translations.
+
+Workflow
+--------
+
+Here are some steps to follow for translating frc-docs.
+
+1. Sign up for `Transifex <https://www.transifex.com/>`__ and ask to join the `frc-docs project <https://www.transifex.com/wpilib/frc-docs>`__, and request access to the language you'd like to contribute to.
+2. Join the wpilibsuite `gitter <https://gitter.im/wpilibsuite/wpilib>`__! This is a direct means of communication with the WPILib team. You can use this to ask us questions in a fast and streamlined fashion.
+3. You may be contacted and asked questions involving contributing languages before being granted access to the frc-docs translation project.
+4. Translate your language!
+
+Links
+-----
+
+Links must be preserved in their original syntax. To translate a link, you can replace the TRANSLATE ME text (this will be replaced with the english title) with the appropriate translation.
+
+Given the following English text, you can
+
+.. code-block:: text
+
+   For complete wiring instructions/diagrams, please see the :doc:`TRANSLATE ME <Wiring the FRC Control System` document>`.
+
+Another example is below
+
+.. code-block:: text
+
+  For complete wiring instructions/diagrams, please see the :ref:`TRANSLATE ME <docs/getting-started/getting-started-frc-control-system/how-to-wire-a-robot:How to Wire an FRC Robot>`
+
+Hebrew
+------
+
+Pronouns
+^^^^^^^^
+
+Pronouns should use **male**. A notice has been applied at the root level index ensuring that the documentation is for all genders.
+
+Publishing Translations
+-----------------------
+
+Translations will be PRed to the `frc-docs <https://github.com/wpilibsuite/frc-docs>`__ on a weekly basis for merge.
+
+Gracious Professionalism
+------------------------
+
+Gracious Professionalism is part of FIRST! Translations must be free of the following
+
+- Slurs
+- Insults
+- Defamation
+- Ungracious Language
```

Comment:
We do not need this section
```suggestion
```

Reply:
I was thinking that we should have something that says that translations should be accurate to the original text. If there's improvements they should be contributed in English and then translated after the PR is merged. 

This covers the same things since there wouldn't be any in English, but doesn't directly call out translators as potentially being bad actors.

Reply:
Love it. Sounds like a plan